src/istgt_md5.c

```diff
@@ -67,7 +67,7 @@ istgt_md5update(ISTGT_MD5CTX *md5ctx, const void *data, size_t len)
 
 	if (md5ctx == NULL)
 		return -1;
-	if (data == NULL || len <= 0)
```

Comment:
Add a comment to the function saying that - len being negative need to be handled by caller.